src/pages/resources/reference/push-notifications/templates/catalog.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Adobe Experience Platform SDK basic push template
+description: Basic push template supported by Adobe Campaign Classic mobile extension.
+keywords:
+- Adobe Campaign Classic
+- Push
+- Push Template
+- Product Catalog
+---
+
+import Tabs from './tabs/catalog.md'
+
+# Push template - product catalog
+
+A notification containing a central image which can show one of three products.<br />
```

Comment:
@calebk1 using a `<br />` tag allows us to make a line break while continuing to add to the "section".  in this case, the first "sentence" under the page title is formatted as a "sub-heading".  two newlines breaks that section, whereas using a `<br />` does not.

in the end it's a style consideration i suppose.  i don't really care which way we choose, as long as we are consistent.

Reply:
Hmm, I see. If we're making this just one header thing, I'd just prefer it to be on one line, since I don't think making these as separate lines necessarily adds very much?